Runtime/KongregatePurchasingModule.cs

```diff
@@ -14,7 +14,7 @@ public override void Configure()
         {
             RegisterStore(
                 KongregateStore.STORE_NAME,
-                (Application.platform == RuntimePlatform.WebGLPlayer && KongregateWeb.IsKongregateAPIAvailable) ? new KongregateStore() : null);
```

Comment:
Now that we have the `KongregateWeb.Status` API, it would be good to also add in an assert that `Status != ApiStatus.Uninitialized`, and either throw an exception or give a warning if the user is trying to register the store without first initializing the web API.